workshops/spinning_wheel/README.md

```diff
@@ -0,0 +1,434 @@
+        <span class="span3"><b>Deadpool</b></span>
+        <span class="span4"><b>Terminator</b></span>
+      </div>
+    </div> 
+    <button class="spin" onclick="myfunction()">SPIN</button>
+  </div>
+  <script src="script.js"></script>
+  </body>
+</html>
+```
+
+Your final code in the index.html should look something like this. Woot woot! We are all done with the HTML part of our project.
+
+## 2)	CSS
+After you have done the writing part of HTML, you'll notice that when you click the "run" button at the top, you just see a white screen with some text("options") with Spin Button. Let's now add some styles to our elements and make them look better like the way we want it to be.🤩
+ 
+Firstly, let’s go to style.css.
+
+Now add the following code:
+
+```css
+*{
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  outline: none;
+}
+
+body{
+  font-family: "Open Sans";
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  overflow: hidden;
+  background: rgb(60, 60, 242);
+  background: linear-gradient(90deg, rgba(60, 60, 242, 1) 0%, rgba(98, 245, 230, 1) 52%, rgba(60, 60, 242, 1) 100%);
+  background-size: cover;
+}
+```
+      
+### Explanation: 
+3 main steps in writing CSS:
+* Select the element/class/id on which you want to apply your styles.
+* Open curly brackets {}.
+* Start adding your styles inside the curly brackets!
+
+In the first line, we have given the size of the box (triangles of our wheel). Then in the second- & third-line padding & margin between them is marked as zero respectively as we don’t require space between the triangles and in fourth line outline is also kept none as we don’t require border when we click on the spin button.
+
+![spin button with border](https://cloud-migxi228s.vercel.app/0spin1.png)
+
+^^^ If we don’t give `outline: none;` then our button will look like this after clicking.
+
+![spin button without border](https://cloud-migxi228s.vercel.app/1spin2.png)
+
+^^^ Whereas if we give `outline: none;` then our button will look like this (without any border) after clicking.
+
+Now in the next block, we have `font-family: Open Sans;` this syntax is used to give the name/type of font we want to have in our block. Here we have used it in selector as body therefore it will be applied as a default font on our whole webpage.
+
+We changed the default display `display: block` to `display: flex`.
+
+A very good reason for this is because flex enables you to align your items perfectly to the center! And that's what we have done in the next two lines. The align-items property aligns all your children elements to center, but horizontally. The justify-content property aligns all your children elements to center, but vertically.
+
+Next, we have the min-height property which sets a minimum height of 100vh (viewport height) to your <body>. The min-height property works together with the align-items property and helps the children to align properly at the center.
```

Comment:
This is fine, though `min-height`, `<body>`, and `align-items` should all be in between backticks